var/spack/repos/builtin/packages/llvm-flang/package.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+import os
+
+
+class LlvmFlang(CMakePackage):
+    """LLVM-Flang is the Flang fork of LLVM needed by the Flang compiler."""
```

Comment:
Might be clearer to reference the Flang package explicitly to help avoid confusion around this being a dependency.
`... needed by the Flang package."""`

Reply:
done